lib/scenic/view.rb

```diff
@@ -42,13 +47,14 @@ def ==(other)
 
     # @api private
     def to_schema
-      materialized_option = materialized ? "materialized: true, " : ""
+      materialized_option = materialized || materialized_no_data ? "materialized: true, " : ""
```

Comment:
Postgres will probably "raise" for us in that case as well, which we may or may not wish to avoid.